<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.core.i18n.NoSuchMessageExceptionSpec" tests="2" skipped="0" failures="0" errors="0" timestamp="2016-12-02T17:43:13" hostname="testing-gce-f9e98b43-93c9-42c8-afdb-eab9b83f6b2a" time="0.005">
  <properties/>
  <testcase name="Exercise constructor with errors" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.005"/>
  <testcase name="Successfully create exception" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
